docs/001_develop/03_client-capabilities/006_charts/002_g2plot-chart/01_chart-types.mdx

```diff
@@ -1,6 +1,6 @@
 ---
 title: 'G2Plot Chart Types'
-sidebar_label: 'Chart types'
+sidebar_label: 'Chart Types'
```

Comment:
As much as I personally like this case, the direction from learn is that we use setence case for titles of pages and sections. Same in all other places
```suggestion
sidebar_label: 'Chart types'
```

Reply:
[chore: addressed PR feedback v1 PTL-1521](https://github.com/genesiscommunitysuccess/docs/pull/1918/commits/7cc3d59862e0ce6d0c1fdb0790b0374f34dbdd8f)